addons/mail/static/src/owl/store/messaging_getters.js

```diff
@@ -12,6 +12,21 @@ const getters = {
     // Public
     //--------------------------------------------------------------------------
 
+    /**
+     * @param {Object} param0
+     * @param {Object} param0.state
+     * @param {integer} id
+     * @return {Object|undefined}
+     */
+    activity({ state }, id) {
+        for (const activityLocalId in state.activities) {
+            let activity = state.activities[activityLocalId];
+            if (activity.id === id) {
+                return activity;
+            }
+        }
+        return undefined;
```

Comment:
shorter and easier to read: `return Object.values(state.activities).map(activity => activity.id === id);`